continuous-integration/linux/build.sh

```diff
@@ -8,12 +8,12 @@ BUILD_ROOT_DIR="$ROOT_DIR/build/ci"
 
 source /etc/os-release || exit $?
 if [[ "$VERSION_ID" == "16.04" ]]; then
-    OS_SPECIFIC_OPTIONS="-DUSE_PCL=OFF -DUSE_EIGEN3=OFF -DUSE_OPENCV=OFF"
+    OS_SPECIFIC_OPTIONS="-DUSE_PCL=OFF -DUSE_EIGEN3=OFF -DUSE_OPENCV=OFF -DUSE_ARUCO=OFF"
 elif [[ "$VERSION_ID" == "18.04" ]]; then
-    OS_SPECIFIC_OPTIONS="-DUSE_EIGEN3=OFF -DUSE_OPENCV=OFF"
+    OS_SPECIFIC_OPTIONS="-DUSE_EIGEN3=OFF -DUSE_OPENCV=OFF -DUSE_ARUCO=OFF"
 elif [[ "$VERSION_ID" == "20.04" ]]; then
     EIGEN3_INCLUDE_DIR="/usr/include/eigen3"
-    OS_SPECIFIC_OPTIONS=""
+    OS_SPECIFIC_OPTIONS="-DUSE_ARUCO=ON"
```

Comment:
I guess this means the OpenCV package on Ubuntu 20.04 comes with the Aruco package?

Why do you have to explicitly set `USE_ARUCO=ON` here, when you don't have to set e.g. `USE_PCL=ON` and `USE_OPENCV=ON` to use those?

Reply:
Oh wait, I saw the cmake file now. This one defaults to `OFF`, but the other default to `ON`. Then you don't need to set them to `OFF` for 16 and 18. But maybe this whole code would be clearer if you explicitly set all of them for all OSes, what do you think?

Reply:
Then the cmake file would have no influence on the build.sh file. Is that what you are suggesting?

Reply:
https://github.com/zivid/zivid-cpp-samples/pull/169/commits/44c55a6492bd9c1bf359b60568dc21765772822a

I can improve the commit later by combining it with the other one where I set these options.

Reply:
> Then the cmake file would have no influence on the build.sh file. Is that what you are suggesting?

I was just thinking it might be easier to read the `build.sh` file if all the `USE_` options are explicitly set for each OS. Then you aren't left wondering what the default is. I see you've done that in a fixup.